example/dummy/dummy.go

```diff
@@ -53,6 +54,18 @@ func (app *DummyApplication) Query(query []byte) types.Result {
 	return types.NewResultOK(wire.JSONBytes(queryResult), "")
 }
 
+func (app *DummyApplication) Proof(key []byte, blockHeight int64) types.Result {
+	if blockHeight != 0 {
+		return types.ErrUnknownRequest
+	}
+	proof, exists := app.state.Proof(key)
+	if !exists {
+		fmt.Println("Didn't find nothing")
```

Comment:
This is why you don't let Americans write English ;)

It's even a double negative, so I guess I did find something...